Assets/Scripts/Control/GameManager.cs

```diff
@@ -46,6 +63,9 @@ public void GameOver(bool won)
             this._stopwatch.Stop();
             Running = false;
 
+            if (!won)
+                this._level = -1;
```

Comment:
Das kannst du dir sparen wenn du das level in dieser Methode hochzählst

Reply:
Das ist der Level Reset wenn man verliert, der muss sein, hochzählen könnte man aber in ner else{lvl++}